addons/website_forum/models/forum_forum.py

```diff
@@ -196,11 +196,11 @@ def _compute_teaser(self):
         for forum in self:
             forum.teaser = textwrap.shorten(forum.description, width=180, placeholder='...') if forum.description else ""
 
-    @api.depends('post_ids')
+    @api.depends('post_ids', 'post_ids.state', 'post_ids.visible')
     def _compute_last_post_id(self):
         for forum in self:
             forum.last_post_id = forum.post_ids.search(
```

Comment:
while we change that method, can we fix the search in the loop ? 
I don't think you can use `_read_group` for that case, but `post_ids` is in the depends, so its fields are in the cache already, we can do something like
```py
domain = [('forum_id', '=', forum.id), ('parent_id', '=', False), ('state', '=', 'active'), ('visible', '=', True)]
forum.last_post_id = min(forum.post_ids.filtered_domain(domain), key=lambda post: post.write_date, default=False)